src/eyepy/__init__.py

```diff
@@ -35,6 +35,7 @@
 from eyepy.core import EyeVolumeLayerAnnotation
 from eyepy.core import EyeVolumeMeta
 from eyepy.core import EyeVolumePixelAnnotation
+from eyepy.io import import_fda
```

Comment:
```suggestion
from eyepy.io import import_topcon_fda
```

I prefer the manufacturer name in the import function name because there might be multiple export formats with the same extension from different manufacturers